rero_ils/modules/documents/dojson/contrib/unimarctojson/model.py

```diff
@@ -482,10 +482,8 @@ def marc21_to_part_of(self, key, value):
         part_of['document'] = {
             '$ref': f'https://bib.rero.ch/api/documents/{linked_pid}'
         }
-        subfield_v = not_repetitive(
-            unimarc.bib_id, 'unimarc', key, value, 'v', default='').strip()
-        if subfield_v:
-            part_of['numbering'] = subfield_v
+        if numbering := utils.force_list(value.get('v'))
+            part_of['numbering'] = numbering
```

Comment:
previously it was a string, now you force as an array : is it correct ?

Reply:
It is even more complex because the `numbering` is an array of an dictionary with "year", "volume", "issue", "pages".
I have to correct this code.